api/app/controllers/spree/api/orders_controller.rb

```diff
@@ -49,8 +49,9 @@ def empty
         render plain: nil, status: 204
       end
 
+      # TODO: this action is never accessible. shall we simply remove it?
```

Comment:
this method was never accessible. why don't we remove it?

Reply:
@coorasse Could you explain why is it unaccessible?

Reply:
it was defined with `:index` and since `:read` was an alias of `:show`, no rule was defined for `:index`. in fact, I replaced it with a random action called `:impossible_action` and no test failed. There is only one test and it checks that this endpoint should not be accessible 🤷‍♂️ . That's why I asked myself: "why don't we remove it?"

Reply:
@coorasse As I've tested, `:manage` gives acces to all rules, also theese not defined, so a user with `can :manage, Order` will go through `authorize! :impossible_action, Order` too. So it could be replaced with `authorize! :admin, Order` I think and it would keep eligible logic.

Reply:
Oh right! `manage` authorizes you! I forgot that 🤦‍♂️ . To do it properly here we should leave `authorize! :index, Order` but then [this test](https://github.com/solidusio/solidus/blob/29fe2cc432ac336ee4dd407f677512c28b35c001/api/spec/requests/spree/api/orders_controller_spec.rb#L201) will fail. So I'd say we could use `authorize! :admin, Order` and keep the current behaviour.